packages/plans-grid/src/plans-interval-toggle/style.scss

```diff
@@ -0,0 +1,112 @@
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/variables';
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/z-index';
+@import '~@automattic/typography/styles/variables';
+@import '~@automattic/onboarding/styles/variables';
+@import '~@automattic/onboarding/styles/mixins';
+@import '../variables';
+
+.plans-interval-toggle {
```

Comment:
In order to satisfy the design spec, I had to:
- disable the `scales/radii` rule (a `border-radius` of `5px` or `6px` is not allowed by stylelint)
- disable the `scales/font-weight` rule (`500`, aka medium font size, is not allowed by stylelint)
- hardcode the colors `#f5f5f5` and `#f52f2f2`(they are not found in [`color-studio`](https://github.com/Automattic/color-studio/blob/master/dist/color-properties.css) )

I'm open to suggestions on how to avoid all of those "exceptions"

Reply:
> * disable the scales/radii rule (a border-radius of 5px or 6px is not allowed by stylelint)
> * disable the scales/font-weight rule (500, aka medium font size, is not allowed by stylelint)

Why do we need to overrule them? What does it add to the UI of the toggle component? Isn't it better to keep consistent and use the default values? 

> * hardcode the colors #f5f5f5 and #f52f2f2(they are not found in color-studio )

Do we have other default values available? We might want to propose using them to stay consistent and align with default styling standards?

Reply:
> Why do we need to overrule them? What does it add to the UI of the toggle component? Isn't it better to keep consistent and use the default values?

It looks like stylelint rules are not up to date with some of the new design specs

>Do we have other default values available? We might want to propose using them to stay consistent and align with default styling standards?

This is probably a question for @ollierozdarz